crates/biome_yaml_parser/src/lexer/tests.rs

```diff
@@ -76,7 +75,7 @@ fn losslessness(string: String) -> bool {
         let mut lexer = YamlLexer::from_str(&cloned);
         let mut tokens = vec![];
 
-        while lexer.next_token(()) != EOF {
+        while lexer.next_token(YamlLexContext::default()) != EOF {
```

Comment:
Is this change correct? I'm getting mixed signals from other occurrences of this test for other languages.

Reply:
Essentially, the change doesn't seem correct. We don't do this in other lexer tests, which means there's something incorrect in the implementation of the lexer

Reply:
Hmmm, I still don't quite understand. The YAML lexer previously didn't use context, but since this PR introduces context into it, `next_token` here also needs to be updated to use the YamlLexerContext. It looks like the other lexers ([html](https://github.com/biomejs/biome/blob/main/crates/biome_html_parser/src/lexer/tests.rs#L24), [css](https://github.com/biomejs/biome/blob/main/crates/biome_css_parser/src/lexer/tests.rs#L26), [js](https://github.com/biomejs/biome/blob/main/crates/biome_js_parser/src/lexer/tests.rs#L81)) are also doing the same?

Reply:
You're right! I misunderstood the message, then probably this shouldn't be a blocker

Reply:
Yup, we're all good here.